app/src/main/kotlin/com/antosik/benchproject/app/favourite/movies/viewModel/FavouriteMoviesViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package com.antosik.benchproject.app.favourite.movies.viewModel
+
+import androidx.lifecycle.ViewModel
+import com.antosik.benchproject.app.common.state.UIState.Empty
+import com.antosik.benchproject.app.common.state.UIState.Success
+import com.antosik.benchproject.app.popular.movies.entity.toUi
+import com.antosik.benchproject.domain.favourite.usecase.GetFavouriteMoviesUseCase
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+import kotlinx.coroutines.flow.flow
+
+@HiltViewModel
+class FavouriteMoviesViewModel @Inject constructor(
+    private val getFavouriteMoviesUseCase: GetFavouriteMoviesUseCase,
+) : ViewModel() {
+
+    val favouriteMoviesViewState = flow {
+        getFavouriteMoviesUseCase().collect { moviesModel ->
```

Comment:
where are tests :D ?

Reply:
I didn't finish this PR, of course, it could be a draft but I forgot to do this. I created this PR for my question and after I finished it (after applying your advice) I will create tests and UI for favourite movies.